tests/unit/test_pyobjects.py

```diff
@@ -64,7 +64,7 @@
 
 
 class Samba(Map):
-    __merge__ = 'samba:lookup'
+    priority = ('os_family', 'os')
```

Comment:
NOTE: `__merge__` is not the correct usage, `merge` would be the correct one to use. However, since we're not testing merging here, I'm simply replacing it with the new `priority` value.